shardingsphere-infra/shardingsphere-infra-authority/shardingsphere-infra-authority-common/src/main/java/org/apache/shardingsphere/authority/provider/schema/builder/SchemaPrivilegeBuilder.java

```diff
@@ -67,7 +67,7 @@
      */
     private static Map<ShardingSphereUser, String> convertSchemas(final String userSchemaMapping) {
         String[] userSchemaPair = userSchemaMapping.trim().split("=");
-        Map<ShardingSphereUser, String> result = new HashMap<>();
+        Map<ShardingSphereUser, String> result = new HashMap<>(1, 1);
```

Comment:
Hi Is there any possibility that it's not 1?

Reply:
Hi, I read the context-related code. This method only processes one userSchemaMapping at a time. Line 76 of the code will put only one element in the map and then execute return. In subsequent processes, only traversal of the map is involved and no element will be put anymore.